core/schema/coremod.go

```diff
@@ -248,6 +248,7 @@ func (m *CoreMod) TypeDefs(ctx context.Context, dag *dagql.Server) ([]*core.Type
 						Description: introspectionArg.Description,
 					}
 
+					// FIXME: allow user-defined local defaults on core functions?
```

Comment:
just flagging a FIXME, fine if out of scope, it does seem a little weird to reason about imo, maybe a step too far?